.swiftlint.yml

```
@@ -70,3 +70,9 @@ custom_rules:
     regex: 'NSLocalizedString([^,]+,\s+comment:\s*"")'
     message: "Localized strings should include a description giving context for how the string is used."
     severity: warning
+
+  string_interpolation_in_localized_string:
+    name: "String Interpolation in Localized String"
+    regex: 'NSLocalizedString\("[^"]+\\\(\S*\)'
```

```suggestion
    regex: 'NSLocalizedString\("[^"]*\\\(\S*\)'
```

This wouldn't catch interpolation at the start of the string:

```swift
NSLocalizedString("\(number) things")
```

---

Well spotted.

---

Wow nice one!